package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@sunbird-rc/verification-sdk",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "An SDK to verify Sunbird generated Verifiable Credentials",
   "main": "./lib/index.js",
   "type": "module",
   "scripts": {
-    "test": "mocha --reporter spec test/*.spec.js"
+    "test": "mocha --require @babel/register --reporter spec test/*.spec.js "
```

Comment:
tests were failing after adding the .js extension. so added babel configuration to run the tests